server/src/config.js

```diff
@@ -274,6 +274,13 @@ var conf = convict({
     default: true,
     env: "DISABLE_SEARCH",
     arg: "disable-search"
+  },
+  staticUrl: {
+    doc: "URL to hose static assets on CDN",
```

Comment:
This doesn't explain enough about the variable.  Maybe something more like "CDN URL prefix, like 'https://somecdn.com/mysite' and links will be rewritten like https://somecdn.com/mysite/static/style.css"

Calling it a CDN might be better, since this isn't where the static files are hosted, but where they are served up –the actual source is still this server.